.github/resources/scripts/cert_manager.py

```
@@ -0,0 +1,271 @@
+        return cert_file
+
+    def get_cluster_server_url(self):
+        """Get the Kubernetes API server URL"""
+        try:
+            result = self.run_command(
+                "kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}'",
+                "Getting cluster server URL",
+                capture_output=True
+            )
+            return result if result else "https://kubernetes.default.svc"
+        except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)